bot/gif_command.go

```diff
@@ -45,13 +45,13 @@ func NewGIFCommand(endpoint, key string, w io.Writer) cli.Command {
 			query := url.Values{}
 			query.Set("key", key)
 			query.Set("q", strings.Join(args, " "))
-			query.Set("limit", "1")
+			query.Set("mediafilter", "minimal")
 			if !c.Bool("explicit") {
 				query.Set("safesearch", "strict")
 			}
 
 			var response TenorSearchResponse
-			if err := client.Get("/v1/random", &response, rclient.Query(query)); err != nil {
+			if err := client.Get("/v1/search", &response, rclient.Query(query)); err != nil {
```

Comment:
since we are switching from random back to search, you should grab a random gif out of the response.

Reply:
I'm not sure, since most people use the !gif almost like a map, where you enter `phrase` and get results, this would break that paradigm. Maybe we could add a random flag instead?

Reply:
I don't think we're on the same page here; nothing would change from the user's perspective. 
My suggestion was geared towards fixing the problem I was originally trying to solve by switching from `/search` to `/random`

Reply:
I think it should change from the user perspective, as in typing a phrase should return the same result.

Reply:
Hmmm, not sure how I feel about that. 
I'm fairly confident everyone who already uses !gif would prefer the random behavior by default; 
However I wouldn't be opposed to adding a `--random` flag and then creating an alias: 
```
!alias add !gif "!gif --random --limit 50"
```